src/site/content/en/tags/index.njk

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout
+title: Tags
+permalink: /en/{{ paged.href }}{% if paged.index > 0 %}{{ paged.index + 1 }}/{% endif %}index.html
+override:tags: []
+description: Our latest news, updates, and stories for developers
+date: 2020-01-30
+renderData:
+  title: "{{ paged.title or title }}"
+  description: "{{ paged.description or description }}"
+pagination:
+  data: collections.paginatedPostsByTag
+  size: 1
+  alias: paged
+---
+
+{# *-landing-page classes are a holdover from devsite. #}
+{# It would be great to remove them as part of the v1 migration. #}
+<div class="blog-landing-page">
```

Comment:
I don't think we have any CSS that targets `blog-landing-page` so we could either remove it, or replace this with `tags-landing-page` in case we ever _do_ want to target this page specifically.